tests/x11/user_defined_snapshot.pm

```diff
@@ -43,7 +43,7 @@ sub run() {
     type_string "yast2 snapper\n";
     assert_screen 'yast2_snapper-snapshots', 100;
     # ensure the last screenshots are visible
```

Comment:
@okurz Yes, that's correct. Will commit fix in upcoming PR, or separate one  in case it takes too long.